src/Nethermind/Ethereum.Test.Base/GeneralTestBase.cs

```diff
@@ -134,7 +126,7 @@ protected EthereumTestResult RunTest(GeneralStateTest test, ITxTracer txTracer)
                 header.ExcessBlobGas = BlobGasCalculator.CalculateExcessBlobGas(parent, spec);
             }
 
-            ValidationResult txIsValid = _txValidator.IsWellFormed(test.Transaction, spec);
+            ValidationResult txIsValid = ChainUtils.ValidateTransaction(test.Transaction, test.Fork);
```

Comment:
I'd prefer not to hardcode this function in `ChainUtils` as it only works for Gnosis and mainnet. Also why use `test.Fork` instead of `spec`
```suggestion
            ValidationResult txIsValid = new TxValidator(test.ChainId).ValidateTransaction(test.Transaction, spec);
```